wagtail/snippets/templates/wagtailsnippets/snippets/edit.html

```diff
@@ -5,6 +5,12 @@
     {% trans "Editing" as editing_str %}
     {% include "wagtailadmin/shared/header_with_locale_selector.html" with title=editing_str subtitle=instance icon="snippet" tabbed=1 merged=1 %}
 
+    {% if latest_log_entry %}
+        <span class="avatar small" data-wagtail-tooltip="{{ latest_log_entry.user_display_name }}"><img src="{% avatar_url latest_log_entry.user size=25 %}" alt="" /></span>
```

Comment:
Unfortunately running low on time to try this manually - will do so later - but does this function without the data-wagtail-tooltip js seen on eg https://github.com/wagtail/wagtail/blob/45dce6ac1e41d13878cc655395757d9ffcba6464/wagtail/admin/templates/wagtailadmin/pages/edit.html?